src/components/WorkHistory/hooks.ts

```diff
@@ -148,8 +152,45 @@ const useUpdateProfileWorkplace = () => {
   return updateProfileWorkplace;
 };
 
+const useFetchAllOrganisations = () => {
+  const accessToken = getAccessToken();
+  const { data } = useQuery(
+    ['all_organisations'],
+    async () => {
+      const response = await fetch(`${config.ApiBaseUrl}/organisation/all`, {
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${accessToken}`,
+        },
+      });
+      const json = await handleApiErrors(response);
+      return json;
+    },
+    {
+      retry: 1,
+      refetchOnWindowFocus: false,
+      onError: (error: any) => {
+        handleError({
+          error,
+          explicitMessage: 'Unable to fetch all Organisation data',
+        });
+      },
+    }
+  );
+
+  const normalizedOrganisationsData = data?.map(
+    (organisation: { name: string; organisationId: number }) => ({
+      label: organisation.name,
+      value: organisation.organisationId,
+    })
+  );
```

Comment:
You can add this after this line -> `const json = await handleApiErrors(response);`